plugins/inputs/unbound/README.md

```diff
@@ -0,0 +1,186 @@
+    total.requestlist.current.all
+    total.requestlist.current.user
+    total.recursion.time.avg
+    total.recursion.time.median
+    time.now
+    time.up
+    time.elapsed
+    mem.total.sbrk
+    mem.cache.rrset
+    mem.cache.message
+    mem.mod.iterator
+    mem.mod.validator
+    histogram.000000.000000.to.000000.000001
+    histogram.000000.000001.to.000000.000002
+    histogram.000000.000002.to.000000.000004
+    histogram.000000.000004.to.000000.000008
+    histogram.000000.000008.to.000000.000016
+    histogram.000000.000016.to.000000.000032
+    histogram.000000.000032.to.000000.000064
+    histogram.000000.000064.to.000000.000128
+    histogram.000000.000128.to.000000.000256
+    histogram.000000.000256.to.000000.000512
+    histogram.000000.000512.to.000000.001024
+    histogram.000000.001024.to.000000.002048
+    histogram.000000.002048.to.000000.004096
+    histogram.000000.004096.to.000000.008192
+    histogram.000000.008192.to.000000.016384
+    histogram.000000.016384.to.000000.032768
+    histogram.000000.032768.to.000000.065536
+    histogram.000000.065536.to.000000.131072
+    histogram.000000.131072.to.000000.262144
+    histogram.000000.262144.to.000000.524288
+    histogram.000000.524288.to.000001.000000
+    histogram.000001.000000.to.000002.000000
+    histogram.000002.000000.to.000004.000000
+    histogram.000004.000000.to.000008.000000
+    histogram.000008.000000.to.000016.000000
+    histogram.000016.000000.to.000032.000000
+    histogram.000032.000000.to.000064.000000
+    histogram.000064.000000.to.000128.000000
+    histogram.000128.000000.to.000256.000000
+    histogram.000256.000000.to.000512.000000
+    histogram.000512.000000.to.001024.000000
+    histogram.001024.000000.to.002048.000000
+    histogram.002048.000000.to.004096.000000
+    histogram.004096.000000.to.008192.000000
+    histogram.008192.000000.to.016384.000000
+    histogram.016384.000000.to.032768.000000
+    histogram.032768.000000.to.065536.000000
+    histogram.065536.000000.to.131072.000000
+    histogram.131072.000000.to.262144.000000
+    histogram.262144.000000.to.524288.000000
+    num.query.type.A
+    num.query.type.PTR
+    num.query.type.TXT
+    num.query.type.AAAA
+    num.query.type.SRV
+    num.query.type.ANY
+    num.query.class.IN
+    num.query.opcode.QUERY
+    num.query.tcp
+    num.query.ipv6
+    num.query.flags.QR
+    num.query.flags.AA
+    num.query.flags.TC
+    num.query.flags.RD
+    num.query.flags.RA
+    num.query.flags.Z
+    num.query.flags.AD
+    num.query.flags.CD
+    num.query.edns.present
+    num.query.edns.DO
+    num.answer.rcode.NOERROR
+    num.answer.rcode.SERVFAIL
+    num.answer.rcode.NXDOMAIN
+    num.answer.rcode.nodata
+    num.answer.secure
+    num.answer.bogus
+    num.rrset.bogus
+    unwanted.queries
+    unwanted.replies
+
+### Tags:
+
+As indicated above, the  prefix of a unbound stat will be used as it's 'section' tag. So section tag may have one of
+the following values:
```

Comment:
It seems like each section is very different, maybe we should call the measurement `unbound_<section>` and remove the tag.  If we did this we would probably want to do something special for the threadX section.

Reply:
As there is not too much data (no really need to spread information in several measurements) and I do not control unbound-control future evolution, I will start to simply drop section/tag and push all fields as they are (just converting field name dots to underscores).  
I will just have a look on the "histograms" fields and see if I drop them or do something intelligible. I am not sure that these fields are really interesting for telegraf as we are collecting timeseries data... we don't really need partially pre-processed historical information.